modules/hibernating-cluster-resume.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/hibernating-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hibernating-cluster-resume_{context}"]
+= Resuming a hibernated cluster
+
+You can resume a hibernated cluster within 90 days. You might have to approve certificate signing requests (CSRs) for the nodes to become ready.
+
+It can take around 45 minutes for the cluster to resume, depending on the size of your cluster.
+
+.Prerequisites
+
+* You hibernated your cluster less than 90 days ago.
++
+// TODO: What if they try to resume later than 90 days? What are the consequences?
+
+* You have access to the cluster as a user with the `cluster-admin` role.
++
+// TODO: Confirm must be a cluster-admin
+
+.Procedure
+
+. Within 90 days of cluster hibernation, resume the cluster virtual machines:
++
+Use the tools native to your cluster's cloud environment to resume the cluster's virtual machines.
+
+. Wait about 5 minutes, depending on the number of nodes in your cluster.
+
+. TODO: Check node status. Sync w/ Vadim on possible script?
+
+. Approve certificate signing requests (CSRs) for the nodes:
+
+.. Check that there is a CSR for each node in the `NotReady` state:
++
+[source,terminal]
+----
+$ oc get csr
+----
++
+.Example output
+[source,terminal]
+----
+NAME       AGE  SIGNERNAME                                   REQUESTOR                                                                  REQUESTEDDURATION  CONDITION
+csr-4dwsd  37m  kubernetes.io/kube-apiserver-client          system:node:ci-ln-812tb4k-72292-8bcj7-worker-c-q8mw2                       24h                Pending
+csr-4vrbr  49m  kubernetes.io/kube-apiserver-client          system:node:ci-ln-812tb4k-72292-8bcj7-master-1                             24h                Pending
+csr-4wk5x  51m  kubernetes.io/kubelet-serving                system:node:ci-ln-812tb4k-72292-8bcj7-master-1                             <none>             Pending
+csr-84vb6  51m  kubernetes.io/kube-apiserver-client-kubelet  system:serviceaccount:openshift-machine-config-operator:node-bootstrapper  <none>             Pending
+----
+
+.. Approve each valid CSR by running the following command:
```

Comment:
@vrutkovs I chose to err on the side of caution of approving CSRs one by one (and then providing the to do them all later on). Let me know if you think that's okay.

Reply:
Good idea, that would be cleaner and equivalent to automated script

Reply:
@vrutkovs Do you think it's still worthwhile to show the alternative for approving all pending CSRs? Or should we just show only the one-by-one option?

Reply:
I ended up deciding to pull out the command to approve all of them.